lib/postcssPlugin.js

```diff
@@ -2,18 +2,31 @@
 "use strict"
 const _ = require("lodash")
 const createStylelint = require("./createStylelint")
-const postcss = require("postcss")
+const postcss/*: Object*/ = require("postcss")
```

Comment:
@davidtheclark , in this particular case, this Object annotation increases flow coverage in this file from 87% to 91%...